exercises/isogram/test/test_isogram.c

```diff
@@ -2,48 +2,60 @@
 #include "../src/isogram.h"
 #include <stdlib.h>
 
+#ifdef RUN_ALL_TESTS
+#define TEST_IGNORE_MESSAGE(X)
```

Comment:
Clever. Does it complain about redefining this?

Reply:
not on my MAC, but Travis doesn't like it.